### DISCUSSION TITLE
I can't view my bucket, web site console show me error message

### DISCUSSION THREAD

**Discussion by n0rmanc** [supabase/supabase#12792] [Questions]
My bucket name is `activitySamples`.
When I click the bucket, I can't view the file list.
I check the console.
It show me the 
```
{
    "message": "id must match /^[a-z0-9-\\.]+$/ regular expression"
}
```

Could any one help me?
![Screenshot 2023-03-03 at 8 56 56 PM](https://user-images.githubusercontent.com/4688217/222725798-42d06691-8d40-4128-980a-0023a332d4f5.png)

**Comment by GaryAustin1** [comment]
![image](https://user-images.githubusercontent.com/54564956/222758843-9747a8e0-eecb-4051-9125-7e89a5507554.png)
